### DISCUSSION TITLE
Need help with uninstalling Poetry

### DISCUSSION THREAD

**Discussion by laughinggas** [python-poetry/poetry#8788] [General] [answered]
Hello,

I have been using Poetry on WSL for a while. This morning I suddenly got an error message saying the command Poetry could not be found. I assumed that Poetry had been deleted from the system somehow. I also got a suggestion to use `sudo apt-get` to install Poetry. I followed the command and realized it was the wrong version (1.1.12), while the lock file I have is version 1.6.1 . I have been trying to uninstall the 1.1.12 version but nothing seems to work. Using `sudo apt-get remove poetry -y` gives me the following error :
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package poetry
```
I later realized that this is not the official way to install Poetry. I would very much appreciate any help I can get to remove this and install the correct version the right way.

Thank you very much!

**Comment by clintonroy** [comment]
I don't know wsl at all, but apt search poetry should give you a hint as to the package name, python3-poetry maybe.

**Reply by laughinggas** [reply]
Thanks! It is indeed python3-poetry. Does that change anything? 
Also, WSL is very similar to Linux.

**Reply by laughinggas** [reply]
This is the output I get :
```
Sorting... Done
Full Text Search... Done
fonts-oflb-euterpe/jammy 1.1-6 all
  unicode musical font

golang-github-knqyf263-go-dep-parser-dev/jammy 0.0~git20190521.1ef8521-2 all
  Golang library for dependency parser

python3-poetry/jammy-updates,now 1.1.12+dfsg-1ubuntu1 all [installed]
  Python dependency management and packaging made easy (Python 3)

python3-poetry-core/jammy,now 1.0.7-2 all [installed,automatic]
  Poetry PEP 517 Build Backend

texlive-humanities/jammy 2021.20220204-1 all
  TeX Live: Humanities packages
```

**Reply by udithapplications** [reply]
I got this output as well but when I try `sudo apt-get remove python3-poetry -y` this is what I get
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package 'python3-poetry' is not installed, so not removed
0 upgraded, 0 newly installed, 0 to remove and 7 not upgraded.
```

**Comment by clintonroy** [comment, marked as answer]
well you should be able to apt remove python3-poetry now...

**Reply by laughinggas** [reply]
That worked, thanks a lot!
